Description: Remove the 'text' feature which relies on the full text
  of licenses, whose copyright status is unclear.
Author: Antonin Delpeuch <antonin@delpeuch.eu>
Forwarded: not-needed
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -46,9 +46,6 @@ repository = "https://github.com/EmbarkS
 [package.metadata.docs.rs]
 all-features = true
 
-[features]
-text = []
-
 [lib]
 name = "spdx"
 path = "src/lib.rs"
